pkg/validate/container.go

```diff
@@ -201,6 +203,117 @@ var _ = framework.KubeDescribe("Container", func() {
 			verifyLogContents(podConfig, logPath, expectedLogMessage)
 		})
 	})
+
+	Context("runtime should support mount propagation", func() {
+		var podID string
+		var podConfig *runtimeapi.PodSandboxConfig
+
+		BeforeEach(func() {
+			podID, podConfig = createPrivilegedPodSandbox(rc, true)
+		})
+
+		AfterEach(func() {
+			By("stop PodSandbox")
+			rc.StopPodSandbox(podID)
+			By("delete PodSandbox")
+			rc.RemovePodSandbox(podID)
+		})
+
+		It("mount with 'rprivate' should not support propagation", func() {
```

Comment:
We should also check mount inside container could not be propagated to host.

Reply:
Will do.